contracts/p1/mixins/RecollateralizationLib.sol

```diff
@@ -190,7 +190,6 @@ library RecollateralizationLibP1 {
             // Ignore dust amounts for assets not in the basket; their value is inaccessible
             // {tok} = {tok/BU} * {BU}
             uint192 inBasket = q.mul(ctx.basketsHeld, FLOOR);
-            if (bal < inBasket) inBasket = bal; // not sure if needed, might be unreachable
```

Comment:
no assert here as well? (like the one in p0 tradinglib?

Reply:
There will still be a revert down below when we do `bal - inBasket`. The assert in P0 is there to search for in our differential testing.